packages/bpk-docs/src/pages/GettingStartedPage/GettingStartedPage.js

```diff
@@ -84,8 +85,21 @@ const components = [
           Backpack for iOS
         </BpkLink>{' '}
         is distributed as a CocoaPod. The pod contains several subspecs for
-        different aspects of Backpack such as color, typography and individual
-        components.
```

Comment:
I've removed the `individual  components` as mentioned while discussing this card